README.md

```diff
@@ -34,17 +34,17 @@ To update the installed JavaScript engines later on, just run `jsvu` again.
 
 ## Supported engines per OS
 
-| JavaScript engine         | Binary name               | `mac64`            | `win32` | `win64`            | `linux32` | `linux64` |
-| ------------------------- | ------------------------- | ------------------ | ------- | ------------------ | --------- | --------- |
-| [**Chakra**][ch]          | `chakra` or `ch`          | ✅                 | ✅      | ✅                 | ❌        | ✅        |
-| [**GraalJS**][graaljs]    | `graaljs`                 | ✅                 | ❌      | ✅                 | ❌        | ✅        |
-| [**Hermes**][hermes]      | `hermes` & `hermes-repl`  | ✅                 | ❌      | ✅                 | ❌        | ✅        |
-| [**JavaScriptCore**][jsc] | `javascriptcore` or `jsc` | ✅                 | ❌      | ✅ <sup>\*</sup>   | ❌        | ✅        |
-| [**QuickJS**][quickjs]    | `quickjs`                 | ❌                 | ✅      | ✅                 | ✅        | ✅        |
-| [**SpiderMonkey**][sm]    | `spidermonkey` or `sm`    | ✅                 | ✅      | ✅                 | ✅        | ✅        |
-| [**V8**][v8]              | `v8`                      | ✅                 | ✅      | ✅                 | ✅        | ✅        |
-| [**V8 debug**][v8]        | `v8-debug`                | ✅                 | ✅      | ✅                 | ✅        | ✅        |
-| [**XS**][xs]              | `xs`                      | ✅ <sup>(32)</sup> | ✅      | ✅ <sup>(32)</sup> | ✅        | ✅        |
+| JavaScript engine         | Binary name               | `mac64`            | `mac64arm`     | `win32` | `win64`            | `linux32` | `linux64` |
+| ------------------------- | ------------------------- | ------------------ | ----------- | ------- | ------------------ | --------- | --------- |
+| [**Chakra**][ch]          | `chakra` or `ch`          | ✅                  | ❌           | ✅      | ✅                 | ❌        | ✅        |
+| [**GraalJS**][graaljs]    | `graaljs`                 | ✅                  | ❌           | ❌      | ✅                 | ❌        | ✅        |
+| [**Hermes**][hermes]      | `hermes` & `hermes-repl`  | ✅                  | ❌           | ❌      | ✅                 | ❌        | ✅        |
+| [**JavaScriptCore**][jsc] | `javascriptcore` or `jsc` | ✅                  | ❌           | ❌      | ✅ <sup>\*</sup>   | ❌        | ✅        |
```

Comment:
I _think_ JSC should already work (but I cannot test it, since I don't have an M1 Mac). But that's unrelated to this patch, so I'll merge this now, and we can update the table later.